src/engine/source/base/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ target_link_libraries(base
     PUBLIC
     fmt::fmt-header-only
     spdlog::spdlog
+    RocksDB::rocksdb
```

Comment:
Do not publish the rocks db library publicly and globally.

Reply:
Changed to the base unit test target, thanks for the advice